### PR TITLE
tidb: add description about `TIDB_CHECK_CONSTRAINTS`

### PR DESCRIPTION
**Pull request by CbcWestwolf**
### What is changed, added or deleted? (Required)

Add description about `TIDB_CHECK_CONSTRAINTS`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions))

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/18753
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lcwangchao — APPROVED** [review]

File: information-schema/information-schema-tidb-check-constraints.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: TIDB_CHECK_CONSTRAINTS
+summary: 了解 INFORMATION_SCHEMA 表 `TIDB_CHECK_CONSTRAINTS`。
+---
+
+# TIDB\_CHECK\_CONSTRAINTS
+
+`TIDB_CHECK_CONSTRAINTS` 表提供关于表上 [`CHECK` 约束](/constraints.md#check-约束)的信息。除了包含 [`CHECK_CONSTRAINTS`](/information-schema/information-schema-check-constraints.md) 中的信息，`TIDB_CHECK_CONSTRAINTS` 还可获取定义 `CHECK` 约束的表名和表 ID。
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-check-constraints.md:8]:
```suggestion
`TIDB_CHECK_CONSTRAINTS` 表提供关于表上 [`CHECK` 约束](/constraints.md#check-约束)的信息。除了包含 [`CHECK_CONSTRAINTS`](/information-schema/information-schema-check-constraints.md) 中的信息，`TIDB_CHECK_CONSTRAINTS` 还提供了定义 `CHECK` 约束的表名和表 ID。
```

File: information-schema/information-schema-tidb-check-constraints.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+title: TIDB_CHECK_CONSTRAINTS
+summary: 了解 INFORMATION_SCHEMA 表 `TIDB_CHECK_CONSTRAINTS`。
+---
+
+# TIDB\_CHECK\_CONSTRAINTS
+
+`TIDB_CHECK_CONSTRAINTS` 表提供关于表上 [`CHECK` 约束](/constraints.md#check-约束)的信息。除了包含 [`CHECK_CONSTRAINTS`](/information-schema/information-schema-check-constraints.md) 中的信息，`TIDB_CHECK_CONSTRAINTS` 还提供了定义 `CHECK` 约束的表名和表 ID。
```
Comment (lilin90) [inline information-schema/information-schema-tidb-check-constraints.md:8]:
```suggestion
`TIDB_CHECK_CONSTRAINTS` 表提供关于表上 [`CHECK` 约束](/constraints.md#check-约束)的信息。除了包含 [`CHECK_CONSTRAINTS`](/information-schema/information-schema-check-constraints.md) 表中的信息之外，`TIDB_CHECK_CONSTRAINTS` 还提供了定义 `CHECK` 约束的表名和表 ID。
```

File: information-schema/information-schema-tidb-check-constraints.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+`TIDB_CHECK_CONSTRAINTS` 表的字段描述如下：
+
+* `CONSTRAINT_CATALOG`：约束的目录，始终为 `def`。
+* `CONSTRAINT_SCHEMA`：约束的库名。
+* `CONSTRAINT_NAME`：约束的名字。
+* `CHECK_CLAUSE`：检查约束的子句。
+* `TABLE_NAME`：约束所在的表名。
+* `TABLE_ID`：约束所在的表 ID。
```
Comment (lilin90) [inline information-schema/information-schema-tidb-check-constraints.md:59]:
```suggestion
* `TABLE_NAME`：约束所在表的名字。
* `TABLE_ID`：约束所在表的 ID。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**